File: src/common/CommandLineCommands/HelpCommand.cs

public class HelpCommand : Command
{
    public HelpCommand()
    {
    }

    public override string GetCommandName()
    {
        return "help";
    }

    public override bool IsEmpty()
    {
        return false;
    }

    public override Task<int> ExecuteAsync(bool interactive)
    {
        throw new NotImplementedException();
    }
}

File: src/commonjs/CommandLineCommands/HelpCommand.ts

import { Command } from '../CommandLine/Command';

export class HelpCommand extends Command {
    public constructor() {
        super();
    }

    public GetCommandName(): string {
        return "help";
    }

    public IsEmpty(): boolean {
        return false;
    }

    public async ExecuteAsync(interactive: boolean): Promise<any> {
        throw new Error("Not implemented");
    }
}